less/serialConsole.less

```diff
@@ -0,0 +1,78 @@
+/**
+  SerialConsole styles
+ */
+
+.console-pf > .terminal {
+  color: @color-pf-white;
+  text-align: left;
+  outline: medium none;
+  background-color: @color-pf-black;
+  border: 1px solid @color-pf-black;
+  padding: 10px;
+}
+
+.console-pf {
+  font-family: @font-family-monospace;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 10px;
+  text-align: center;
+  line-height: normal;
+}
+
+@media (min-width: @screen-tablet) {
+  .console-pf {
+    font-size: @font-size-small;
+  }
+}
+
+.console-pf > pre {
```

Comment:
I'm not finding a <pre> anywhere. Is this needed?

Reply:
Done, thanks